crates/engine/src/execution/request/mod.rs

```diff
@@ -33,27 +33,27 @@ impl<R: Runtime> Engine<R> {
     pub(crate) fn unpack_http_request<B>(
         &self,
         request: http::Request<B>,
-    ) -> Result<(EarlyHttpContext, http::HeaderMap, B), http::Response<Body>> {
+    ) -> Result<(EarlyHttpContext, http::HeaderMap, B), Box<http::Response<Body>>> {
```

Comment:
Why this change?

Reply:
If it's because the error variant is larger than the ok variant, it's generally not true, it's not taking `B` into account, which will pretty much always be the same size as the default Body.

Reply:
Yeah it's clippy not letting us to do that. Should we just say clippy to go somewhere else?